tests/AddAndFind_query.json

```diff
@@ -190,7 +190,7 @@
          "Age":  [">", 0, "<=", 100 ]
       },
         "results": {
-            "list":["Name","Age","Email"],
+            "list":["Name","Age","Email", "Study"],
```

Comment:
Adding this is not checking whether the response is correct. 
We should add a tests that checks whether we are getting the right response. 

I think we should move this tests to python, instead of using this file and C++ (not for this, but for another different pull request for later)

Reply:
Yeah I did my usual thing of visual inspection 😀